endpoints/events/vtrack.go

```diff
@@ -69,6 +70,13 @@ func (v *vtrackEndpoint) Handle(w http.ResponseWriter, r *http.Request, _ httpro
 		return
 	}
 
+	// get integration value from request parameter
+	integrationType, err := getIntegrationType(r)
+	if err != nil {
+		w.Write([]byte(fmt.Sprintf("Invalid integration type: %s\n", err.Error())))
```

Comment:
Please add the `http.StatusBadRequest` response code when returning an error here. Look at L85 for reference